godot-core/src/task/futures.rs

```diff
@@ -263,7 +263,7 @@ impl<R: ParamTuple + IntoDynamicSend> Future for FallibleSignalFuture<R> {
 impl<R: ParamTuple + IntoDynamicSend> Drop for FallibleSignalFuture<R> {
     fn drop(&mut self) {
         // The callable might alredy be destroyed, this occurs during engine shutdown.
-        if self.signal.object().is_none() {
+        if self.signal.is_null() {
```

Comment:
What motivates this particular change? If `is_null` is safer, would it warrant a comment here (also to prevent potential reverts in the future)?

Reply:
It's fine, if we can keep it simpler, that's good! Just wondered if there's a difference :slightly_smiling_face: